app/pages/sections/supplier/supplier/contextCreator.test.js

```diff
@@ -0,0 +1,38 @@
+import manifest from './manifest.json';
+import { getContext } from './contextCreator';
+import { baseUrl } from '../../../../config';
+
+describe('supplier contextCreator', () => {
+  describe('getContext', () => {
+    it('should return the backLinkText', () => {
+      const context = getContext({ orderId: 'order-1' });
+      expect(context.backLinkText).toEqual(manifest.backLinkText);
+    });
+
+    it('should construct the backLinkHref', () => {
+      const orderId = 'order-id';
+      const context = getContext({ orderId });
+      expect(context.backLinkHref).toEqual(`${baseUrl}/organisation/${orderId}/supplier/search/select`);
+    });
+
+    it('should return the title', () => {
```

Comment:
this isn't the usual format of 'should return the contents of the manifest'?

Reply:
I've been explicitly checking each field. I think think contextCreator is going to start getting pretty complex soon to handle various different states for this page